### PR TITLE
chore:Prep release 1.12.0

### PR DESCRIPTION
**Pull request by pankajagrawal16**
This is automated release prep. Remember to update [CHANGELOG.md](https://github.com/awslabs/aws-lambda-powertools-java/blob/prep-release-1.12.0/CHANGELOG.md) to capture changes in this release. Please review changes carefully before merging.

* [ ] Updated CHANGELOG.md

### CODE REVIEW

**Comment by pankajagrawal16** [comment]
cc @jeromevdl , Can you please update the changelog?

**Comment by jeromevdl** [comment]
### Added
 * **Easy Event Deserialization**: Extraction and deserialization of the main content of events (body, messages, ...) #757

### Bug Fixes
 * Different behavior while using SSMProvider with or without trailing slash in parameter names #758


I don't know if there is something else? @pankajagrawal16

**Comment by pankajagrawal16** [comment]
I think these are the relevant one.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ This project follows [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) fo
 
 ## [Unreleased]
 
+## [1.12.0] - 2022-03-01
+
+<PLEASE REMEBER TO UPDATE CHANGE LOG>
```
Comment (jeromevdl) [inline CHANGELOG.md:13]:
```suggestion
### Added
 * **Easy Event Deserialization**: Extraction and deserialization of the main content of events (body, messages, ...) #757

### Bug Fixes
 * Different behavior while using SSMProvider with or without trailing slash in parameter names #758
```

**Review by jeromevdl — DISMISSED** [review]

**Pull request merged**